src/index.ts

```diff
@@ -0,0 +1,124 @@
+class School {
+  directions: Direction[] = [];
+
+  addDirection(direction: Direction) {
+    this.directions.push(direction);
+  }
+}
+
+class Direction {
+  levels: Level[] = [];
+  private _name: string;
+
+  get name(): string {
+    return this._name;
+  }
+
+  constructor(name: string) {
+    this._name = name;
+  }
+
+  addLevel(level: Level) {
```

Comment:
Please, do not forget to add explicitly type annotation for the function return type.